kong/runloop/balancer.lua

```diff
@@ -908,6 +908,7 @@ local function execute(target, ctx)
                                           hash_value)
     if not ip and
       (port == "No peers are available" or port == "Balancer is unhealthy") then
+      log(ERR, "failed to get a peer from the ring-balancer (", port, ")")
```

Comment:
I don't think this is correct. This is not a "code error", but a user(=admin) error, hence it should not be reported through the system-logs, but through a traffic-log (aka a log plugin). Also because this would in a high-load system fill up the error logs.

Reply:
The intent was clear. This was useful when debugging the balancer over the last few days and could be useful when receiving future error logs for review around balancer issues.

Reply:
Not sure what you mean by that? want to keep it in, or should it go?

Reply:
I don't have strong feelings. Do you?

Reply:
I don't want it, because on a busy system you'll get hundreds of these errors if there is a mistake. Hence I think they belong in the traffic logs, and not system logs.

I'll remove it

Reply:
For us though, it'd still be helpful to distinguish between the two possible errors here. We should consider slightly updating the error message to help making that distinction.

Reply:
Merging as-is for inclusion in 2.0.4 - we can add such a distinction later.